src/supertux/sector.cpp

```diff
@@ -713,10 +786,21 @@ Sector::get_camera() const
   return get_singleton_by_type<Camera>();
 }
 
-Player&
-Sector::get_player() const
+std::vector<Player*>
+Sector::get_players() const
 {
-  return *static_cast<Player*>(get_objects_by_type_index(typeid(Player)).at(0));
+  auto players_raw = get_objects_by_type<Player>();
+
+  std::vector<Player*> players;
+
+  auto it = players_raw.begin();
+  while (it != players_raw.end())
```

Comment:
Can you not use `std::transform` for this?

Reply:
The iterator used here (`it`) is our own implementation of an iterator-*like* object; it lacks many functions like `get()` or others. I can try to see if it works here.

Also, IMO `std::transform` is harder to understand here, since there's no actual "transform"; is there some copy-only equivalent?